NEASQC/WP6_QNLP
# unifies bash files for step 6

## Scope

- Combines step 6 bash files for beta and pre alphas into one file

## Implementation

- Combined code in one file
- Added appropriate conditional statements and command line arguments
- 
## How to Test

Run bash file specifying "beta_quantum" for the model tag to run beta; train, test, labels and k must also be specified. Pre alpha commands should work as before.

## Emoji Guide
**For reviewers: Emojis can be added to comments to call out blocking versus non-blocking feedback.**
E.g: Praise, minor suggestions, or clarifying questions that don’t block merging the PR.
> 🟢 Nice refactor!
> 🟡 Why was the default value removed?
E.g: Blocking feedback must be addressed before merging.
> 🔴 This change will break something important

| | | |
| --- | --- | --- |
| Blocking | 🔴 ❌ 🚨 | Red |
| Non-blocking | 🟡 💡 🤔 💭 | Yellow, thinking, etc |
| Praise | 🟢 💚 😍 👍 🙌 | Green, hearts, positive emojis, etc |
